lib/fetch/headers.js

```diff
@@ -55,8 +71,10 @@ function fill (headers, object) {
 
     // 2. Otherwise, object is a record, then for each key → value in object,
     //    append (key, value) to headers
-    for (const [key, value] of Object.entries(object)) {
-      headers.append(key, value)
+    const keys = Object.keys(object)
+    for (let i = 0; i < keys.length; i++) {
+      const key = keys[i]
+      headers.append(key, object[key])
```

Comment:
```suggestion
    const entries = Object.entries(object)
    for (let i = 0; i < entries.length; i++) {
      headers.append(entries[i][0], entries[i][1])
```

remove unnecessary `const` declaration

Reply:
Object.entries is very slow in v8

Reply:
Good to know. I wonder what the diff is between the declaration here. Maybe just removing that in favor of inlining `keys[i]` then

Reply:
Hello!
I have removed the variables.